check/filesystem.lua

```diff
@@ -136,4 +144,18 @@ function FileSystemCheck:run(callback)
   callback(checkResult)
 end
 
+function GetOptionsStringForFs(mountPoint)
+  local procMountsFilename = env.get('TEST_PROC_MOUNTS') or '/proc/mounts'
+  local procMountsData, err = lfs.readFileSync(procMountsFilename)
+  if err then return nil end
+  for line in procMountsData:gmatch"(.-)\n" do
```

Comment:
Neat, I didn't know about the `-` operator.  Looks like a non-greedy version of `*`.